tests/Doctrine/Tests/DbalTypes/CustomIdObjectType.php

```diff
@@ -14,15 +14,15 @@ class CustomIdObjectType extends Type
      */
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
-        return $value->id;
+        return $value->id . '_test';
```

Comment:
Please use new test assets for your tests: modifying existing test assets is extremely risky.